tests/core_tests/wallet_tools.cpp

```diff
@@ -67,11 +79,12 @@ void wallet_tools::process_transactions(tools::wallet2 * wallet, const std::vect
     wallet_tools::gen_block_data(bt, bl, mtx, v_bche.back(), v_parsed_block.back(), idx == 1 ? start_height : height);
```

Comment:
Not a bug that was introduced in this PR, but if this loop iterates more than once, it will use `height` which is never initialized.

Reply:
in `gen_block_data` the height parameter is `uint64_t &height` and it set immediately (out param), it is ok then, no?

Reply:
Ah, yes, I didn't notice it. Then it's ok.